### PR TITLE
change licence information, add it to missing files

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #214
| License       | MIT

### CODE REVIEW

**Comment by ElectricMaxxx** [comment]
btw: that is a good chance to detect files, which have no licence header. In my case: the ones, that are not blue.

File: DependencyInjection/CmfSeoExtension.php [hunk 1]
```diff
@@ -11,13 +11,12 @@
...
 namespace Symfony\Cmf\Bundle\SeoBundle\DependencyInjection;
 
+use Symfony\Cmf\Bundle\RoutingBundle\Routing\DynamicRouter;
 use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
-use Symfony\Cmf\Bundle\RoutingBundle\Routing\DynamicRouter;
```
Comment (ElectricMaxxx) [inline DependencyInjection/CmfSeoExtension.php]: These line changes are really strange? I didn't make them.

**Comment by ElectricMaxxx** [comment]
there are lots of changes i do not understand. I didn't ever touched stuff like `use` sections.

File: DependencyInjection/CmfSeoExtension.php (hunk 1 above)
Comment (dbu) [inline DependencyInjection/CmfSeoExtension.php]: did you use something like php-cs-fixer ? as the build is still green, i think this is just removing unused use statements and sorting them alphabetically.

**Comment by dbu** [comment]
:+1:

**Pull request merged**